_includes/releases/v22.1/v22.1.0-alpha.4.md

```diff
@@ -0,0 +1,97 @@
+## v22.1.0-alpha.4
+
+Release Date: March 21, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-alpha-4-security-updates">Security updates</h3>
+
+- Clusters can be configured to send HSTS headers with HTTP requests in order to enable browser-level enforcement of HTTPS for the cluster host. This is controlled by setting the `server.hsts.enabled` [cluster setting](../v22.1/cluster-settings.html) to `true` (default: `false`). Once the headers are present, after an initial request, browsers will force HTTPS on all subsequent connections to the host. This reduces the possibility of man-in-the-middle (MitM) attacks, which HTTP-to-HTTPS redirects are vulnerable to. [#77244][#77244]
+
+<h3 id="v22-1-0-alpha-4-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Added a `created` time column to the `crdb_internal.active_range_feeds` virtual table to improve observability and debuggability of the rangefeed system. [#77597][#77597]
+- Incremental backups created by [`BACKUP ... INTO`](../v22.1/backup.html) or [`BACKUP ... TO`](../v22.1/backup.html) are now stored by default under the path `incrementals` within the backup destination, rather than under each backup's path. This enables easier management of cloud-storage provider policies specifically applied to [incremental backups](../v22.1/take-full-and-incremental-backups.html#incremental-backups). [#75970][#75970]
```

Comment:
Super minor: I would maybe flip the link to the first instance of "Incremental backups" here at the start of the sentence. `/incrementals` might be more descriptive, since we're discussing path. It sits at the root of the directory. Also, I could see this just being left to me to describe further in the docs :-)